src/openapi.yaml

```diff
@@ -70,6 +70,19 @@ paths:
     get:
       operationId: zaakinformatieobject_list
       description: Beheer relatie tussen InformatieObject en ZAAK.
+      parameters:
+        - name: zaak
+          in: query
+          description: ''
+          required: false
+          schema:
+            type: string
```

Comment:
ik mis hier `format: url` - deze zou voor `URLField` of `URLModelChoiceField` moeten gezet worden automatisch.